### PR TITLE
Adding snyk check to this repo

### PR DESCRIPTION
**Pull request by rhiananthony**
### Purpose
Addition of snyk check to this repo
---
### Changes
Adding section to Readme file to indicate Snyk check status

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] This PR applied Python style guidelines, specifically followed [NumPy style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) for this repo.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -11,6 +11,10 @@ Single Cell Tools
   :target: https://sctools.readthedocs.io/en/latest/?badge=latest
   :alt: Documentation Status
 
+.. image:: https://img.shields.io/snyk/vulnerabilities/github/HumanCellAtlas/lira/requirements.txt.svg?label=Snyk%20Vulnerabilities&logo=Snyk
```
Comment (samanehsan) [inline README.rst]: Should this link be to `sctools` instead of `lira`?

**Comment by samanehsan** [comment]
Why is there an internal error with pullapprove? 🤔

File: README.rst (hunk 1 above)
Comment (rhiananthony) [reply README.rst]: Oops. Yes. Fixing :)

Comment (rhiananthony) [reply README.rst]: Fixed

**Pull request merged**